components/eam/src/physics/cam/hetfrz_classnuc.F90

```diff
@@ -38,10 +38,46 @@ module hetfrz_classnuc
 real(r8) :: rhoh2o
 real(r8) :: mwh2o
 real(r8) :: tmelt
-real(r8) :: pi
+!real(r8) :: pi     ! commented out to avoid double declaring pi
```

Comment:
why not removing this line?

Reply:
removed this line, and added comments for the one used in the module